pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/argoproj.io/v1alpha1/Workflow/testdata/interpretdependency-test.yaml

```diff
@@ -0,0 +1,85 @@
+# test case for interpreting dependency of Workflow
+
```

Comment:
You can list the test cases included in the current file at the beginning of the document. This allows readers to have a comprehensive overview of the tests included in the current document.